debian/rules

```diff
@@ -107,8 +104,8 @@ endif
 	    -DCONC_DEFAULT_CHARSET=utf8mb4 \
 	    -DPLUGIN_AWS_KEY_MANAGEMENT=NO \
 	    -DPLUGIN_COLUMNSTORE=NO \
-	    -DIGNORE_AIO_CHECK=YES \
-	    -DWITH_URING=YES \
+	    -DIGNORE_AIO_CHECK=ON \
+	    -DWITH_URING=ON \
```

Comment:
Same as above

Reply:
This just changes `YES` to `ON` to unify the terms. There are no functional changes.